drivers/docker/driver.go

```diff
@@ -929,6 +930,29 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 	} else {
 		// TODO add support for more than one network
 		network := task.Resources.NomadResources.Networks[0]
+		nebulaIntfName := "nebula1"
```

Comment:
Do you think it's worth taking this as an env var in case this interface changes?

Reply:
I know that 0.12.x should support this (https://github.com/hashicorp/nomad/pull/8208/commits). But it doesn't appear  that way from looking in the src in master: https://github.com/hashicorp/nomad/blob/master/drivers/docker/driver.go#L1040-L1041

Reply:
> Do you think it's worth taking this as an env var in case this interface changes?

Sure, how hard is it to pass in env variable to the nomad client?

Reply:
```
v0.11.3
build: Updated to Go 1.14.3 [GH-7431]
csi: Return better error messages [GH-7984] [GH-8030]
csi: Move volume claim releases out of evaluation workers [GH-8021]
csi: Added support for VolumeContext and VolumeParameters [GH-7957]
driver/docker: Added support for memory_hard_limit configuration in docker task driver [GH-2093]
logging: Remove spurious error log on task shutdown [GH-8028]
ui: Added filesystem browsing for allocations [GH-5871]

v0.11.2
core: Added support for a per-group policy to stop tasks when a client is disconnected [GH-2185]
core: Allow spreading allocations as an alternative to binpacking [GH-7810]
client: Improve AWS CPU performance fingerprinting [GH-7681]
csi: Added support for volume secrets [GH-7923]
csi: Added periodic garbage collection of plugins and volume claims [GH-7825]
csi: Improved performance of volume claim releases by moving work out of scheduler [GH-7794]
driver/docker: Added support for custom runtimes [GH-7932]
ui: Added ACL-checking to conditionally turn off exec button [GH-7919]
ui: Added ACL-checking to conditionally turn off exec button [GH-7919]
ui: CSI searchable volumes and plugins pages [GH-7895]
ui: CSI plugins list and etail pages [GH-7872] [GH-7911]
ui: CSI volume constraints table [GH-7872]

v0.11.1
only bug fixes


v0.11.0
Container Storage Interface [beta]: Nomad has expanded support
of stateful workloads through support for CSI plugins.
Exec UI: an in-browser terminal for connecting to running allocations.
Audit Logging (Enterprise): Audit logging support for Nomad
Enterprise.
Scaling APIs: new scaling policy API and job scaling APIs to support external autoscalers
Task Dependencies: introduces lifecycle stanza with prestart and sidecar hooks for tasks within a task group


v0.11.0-rc1
connect: Added support for using Connect with TLS enabled Consul agents [GH-7602]
connect: Fixed a bug where restarting a client would prevent Connect enabled allocations from cleaning up properly [GH-7643]
consul: Fixed a bug where service updating could sometimes fail [GH-7600]
scheduler: Fixed a bug where changes to task group shutdown_delay were not persisted or displayed in plan output [GH-7618]
A number of documentation updates and test improvements.

v0.11.0-beta2
connect: Added support for specifying Envoy expose path configurations
cli: Fixed a bug where nomad agent -dev fails on Windows
client: Fixed a bug affecting network detection in environments that mimic the EC2 Metadata API
driver/docker: Fixed handling of seccomp security_opts option
driver/docker: Fixed a bug causing docker containers to use swap memory unexpectedly
ui: Fixed handling of multi-byte unicode characters in allocation log view
ui: Fixed exec outside test environment
ui: Removed exec button for dead jobs
ui: Added exec-specific loading template
api: change to the /v1/job/:id/scale API [beta]


v0.11.0-beta1
Container Storage Interface [beta]: Nomad has expanded support
of stateful workloads through support for CSI plugins.
Exec UI [beta]: an in-browser terminal for connecting to running allocations.
Audit Logging (Enterprise) [beta]: Audit logging support for Nomad
Enterprise.
Scaling APIs [beta]: new scaling policy API and job scaling APIs to support external autoscalers
Task Dependencies: introduces lifecycle stanza with prestart and sidecar hooks for tasks within a task group

```

Reply:
Nomad client runs as a system service so think we could just inject it via systemd unit file. And.. yeah it wouldn't show up in an upstream changelog since this is your code?

Reply:
Yeah i was checking for something in the change log. While doing so, I took a peek at the latest version (0.12.x) of the file I'm changing in this MR and it didn't look like it supports multiple interfaces the way we want to... but of course I'm taking an extremely narrow view of the whole system. The logic is probably elsewhere :)